docs/FAQ.md

```diff
@@ -563,6 +561,31 @@ Usually, the different commits associated with the conflicted bookmark should al
 appear in the log, but if they don't you can use `jj bookmark list`to show all the
 commits associated with it.
 
+### How can I avoid accidentally tracking new files I didn't mean to track?
+
+Users with habits from a different VCS like Git may be used to new files being explicitly tracked.
+To them, it can be surprising that newly added files are "silently" tracked in the working copy.
+In the worst case scenario, files with sensitive information are accidentally pushed to a public repository.
+
+You can configure jj to [not automatically track certain files](config/#paths-to-automatically-track).
```

Comment:
Fix this link. It should probably be something like `config.md#paths-to-automatically-track`.